discord/ext/commands/parameters.py

```diff
@@ -152,6 +158,11 @@ def converter(self) -> Any:
 
         return self.annotation
 
+    @property
+    def description(self) -> Optional[str]:
+        """Optional[:class:`str`]: The description of this parameter."""
+        return self._description if self._description is not empty else None
```

Comment:
This shouldn't be empty right?

```suggestion
        return self._description if self._description is not MISSING else None
```

Reply:
Sorry please ignore this